docs/podman-search.1.md

```diff
@@ -13,7 +13,9 @@ The user can specify which registry to search by prefixing the registry in the s
 **registires.search** table in the config file - **/etc/containers/registries.conf**.
 The number of results can be limited using the **--limit** flag. If more than one registry
 is being searched, the limit will be applied to each registry. The output can be filtered
-using the **--filter** flag.
+using the **--filter** flag. To get all available images in a registry without a specific
+query, the user can just enter the registry name with a trailing (example **registry.fedoraproject.org/**).
```

Comment:
No we don't allow multiple registries to be handed in. If you just do `podman search image` then it will try all the registries from your registries.conf file. If you do `podman search docker.io/image` it will only search docker.io for image. And now if you do `podman search myregistry/`, it will return all the images it finds in the registry if it is a v2 registry.